testing/jormungandr-integration-tests/src/common/jormungandr/rest.rs

```diff
@@ -10,18 +16,24 @@ pub enum RestError {
     CannotDeserialize(#[from] serde_json::Error),
     #[error("could not send reqeuest")]
     RequestError(#[from] reqwest::Error),
+    #[error("hash parse error")]
+    HashParseError(#[from] chain_crypto::hash::Error)
 }
 
 /// Specialized rest api
-#[derive(Debug)]
+#[derive(Debug,Clone)]
```

Comment:
add some spacing in your derive please
```suggestion
#[derive(Debug, Clone)]
```